KoreBuild-dotnet/build/_dotnet-test.shade

```diff
@@ -18,7 +18,16 @@ default configuration = 'Debug'
 
 @{
     var projectFolder = Path.GetDirectoryName(projectFile);
-    var testArgs = "--configuration " + configuration;
-    Dotnet("test " + testArgs, projectFolder);
+    var projectName = Path.GetFileName(projectFolder);
+
+    var noParallelTestProjects = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
+    if (!string.IsNullOrEmpty(NO_PARALLEL_TEST_PROJECTS))
```

Comment:
Will probably need to restore the `NO_PARALLEL_TEST_PROJECTS` setting in at least the MVC repo.